### DISCUSSION TITLE
v0.2.6

### DISCUSSION THREAD

**Discussion by tomakino** [tomakino/CSLyric#4] [Announcements]
### 新增支持
 - 酷狗音乐概念版

### 新功能
 - 可在锁屏页面隐藏歌词
 - 可调整歌词无限滚动
 - API支持
 
### 其他
软件已上传到Xposed模块仓库
https://github.com/Xposed-Modules-Repo/com.makino.cslyric

<hr /><em>This discussion was created from the release <a href='https://github.com/tomakino/CSLyric/releases/tag/v0.2.6'>v0.2.6</a>.</em>

**Comment by masonia2** [comment]
![Screenshot_2024-10-17-20-27-30-223_lockscreen](https://github.com/user-attachments/assets/478db24d-4019-40ca-9be4-a485b3bced5a)
有bug，锁屏界面左上角时间不会隐藏
